notebooks/dataframes/integrations.ipynb

```diff
@@ -0,0 +1,445 @@
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "table_id = df.to_gbq(clustering_columns=(\"index\", \"int_col\"))\n",
+    "table_id"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Returning a BigQuery DataFrames (bigframes) DataFrame\n",
+    "\n",
+    "The recommended way to construct a DataFrame is from a BigQuery table which has a unique primary key. By default a primary key is used as the index, which allows for more efficient queries than the default index generation.\n",
+    "\n",
+    "This sample assumes there is a shared dataset that\n",
+    "\n",
+    "1. The application can write to and\n",
+    "2. the bigframes user can read from.\n",
+    "\n",
+    "There are many ways an application can [write to a BigQuery table](https://cloud.google.com/bigquery/docs/loading-data), including BigQuery load jobs, DML, streaming REST API, and the BigQuery Write API. Each has different costs, performance, and limitations. Choose the one that best suits your application's needs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The assumption is that there is a shared dataset to work with.\n",
+    "from google.cloud import bigquery\n",
+    "\n",
+    "bqclient = bigquery.Client()\n",
+    "bqclient.create_dataset(\"my_dataset\", exists_ok=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# For simplicity, this sample assumes your application uses\n",
+    "# a load job with the CSV file format.\n",
+    "# See: https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-csv#python\n",
+    "\n",
+    "def create_table_for_bigframes():\n",
+    "    # This code is assumed to run on the application's backend.\n",
+    "    from google.cloud import bigquery\n",
+    "\n",
+    "    client = bigquery.Client()\n",
+    "    job_config = bigquery.LoadJobConfig(\n",
+    "        schema=[\n",
+    "            bigquery.SchemaField(\"name\", \"STRING\"),\n",
+    "            bigquery.SchemaField(\"post_abbr\", \"STRING\"),\n",
+    "        ],\n",
+    "        skip_leading_rows=1,\n",
+    "        source_format=bigquery.SourceFormat.CSV,\n",
+    "    )\n",
+    "    uri = \"gs://cloud-samples-data/bigquery/us-states/us-states.csv\"\n",
+    "\n",
+    "    load_job = client.load_table_from_uri(\n",
+    "        uri, table_id, job_config=job_config\n",
+    "    )  # Make an API request.\n",
+    "\n",
+    "    load_job.result()  # Waits for the job to complete.\n",
+    "\n",
+    "    destination_table = client.get_table(table_id)  # Make an API request.\n",
+    "    print(\"Loaded {} rows.\".format(destination_table.num_rows))\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
```

Comment:
Is this empty cell (line 416-422) intended here?

Reply:
Good catch. I've removed the empty cells and made an update to the "Returning a BigQuery DataFrames (bigframes) DataFrame" sample.